app/helpers/feedex_helper.rb

```diff
@@ -23,4 +23,12 @@ def get_response_total(total_count, results_limited, scopes)
     response_total = "Over #{response_total}" if results_limited
     response_total
   end
+
+  def confirmation_message(total_count)
+    if total_count && total_count > 1000
+      "A lot of records are about to be exported: #{total_count}. Are you sure you to export this list?"
+    else
+      "This will export #{total_count} records. Are you sure you want to export this list?"
+    end
```

Comment:
There's not much difference between these two messages, could we just have one message?  Or, always have "This will export <blah> records. Are you sure?" but add "This is a large number of records and may take some time." for the large case.